src/go/intelligence/config.md

```diff
@@ -213,9 +217,20 @@ YOU ARE FREE TO SET VALUES FOR NEW SERVICES THOUGH.
         "down": "urgent"
       },
       "annotationMap": {
-        # "label": "shortDescription",              #DO NOT CHANGE IT, UNIQUE FOR SSB
-        "actions": ["intervention"],
-        "systems": ["network", "database", "db"],
+        "label": "shortDescription",               #DO NOT CHANGE IT, UNIQUE FOR SSB
+        "annotations": [
+          {
+            "actions": ["intervention", "outage"],
+            "systems": ["network", "database", "db"],
+            "tags": ["cmsweb", "das"]
+          },
+
+          {
+            "actions": ["update", "upgrade"],
+            "systems": ["network", "database", "db"],
+            "tags": ["jobs"]
```

Comment:
I saw above tags with prod and jobs, should we have them here too?

Reply:
Not actually. This is just an example. Giving an overview of the config file.